services/movie_session.py

```diff
@@ -24,6 +24,16 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = MovieSession.objects.get(
+        id=movie_session_id
+    ).tickets.values("row", "seat")
+    return [
+        {"row": ticket["row"], "seat": ticket["seat"]}
+        for ticket in tickets
+    ]
```

Comment:
you actually do the same job twice using .values() and creating a list of dicts.
.values() returns queryset of dicts. you can use this fact